core/tests/protractor_utils/PreferencesPage.js

```diff
@@ -107,9 +107,13 @@ var PreferencesPage = function() {
   };
 
   this.setUserBio = async function(bio) {
+    await waitFor.visibilityOf(
+      userBioElement, 'User bio field takes too long to appear.');
     await userBioElement.clear();
     await userBioElement.sendKeys(bio);
     await navBar.click();
```

Comment:
Maybe in any other PR we should also wait for this element to be clickable?

Reply:
I did consider that, but I felt it was not required because the navBar element refers to the dropdown in the top right corner. The only condition where this won't be present is when the user has not logged in. In which case, it would fail in line 110 (or much earlier, in the calling function).